### PR TITLE
interfaecs/apparmor: remove all traces of old-security from apparmor backend

### PR DESCRIPTION
**Pull request by zyga**
This branch streamlines the apparmor policy code that no longer needs to support old-security

### CODE REVIEW

File: interfaces/apparmor/template.go [hunk 1]
```diff
@@ -245,17 +245,17 @@ var defaultTemplate = []byte(`
...
-  /{dev,run}/shm/snap/@{APP_PKGNAME}/@{APP_VERSION}/** mrwlkix,
+  /{dev,run}/shm/snap/@{SNAP_NAME}/                  r,
+  /{dev,run}/shm/snap/@{SNAP_NAME}/**                rk,
+  /{dev,run}/shm/snap/@{SNAP_NAME}/@{SNAP_REVISION}/   r,
+  /{dev,run}/shm/snap/@{SNAP_NAME}/@{SNAP_REVISION}/** mrwlkix,
 
   # Allow apps from the same package to communicate with each other via an
   # abstract or anonymous socket
-  unix peer=(label=@{APP_PKGNAME}_*),
+  unix peer=(label=@{SNAP_NAME}_*),
```
Comment (jdstrand) [inline interfaces/apparmor/template.go]: This needs to be:
`+  unix peer=(label=snap.@{SNAP_NAME}.*),`

File: interfaces/apparmor/template.go [hunk 2]
```diff
@@ -245,17 +245,17 @@ var defaultTemplate = []byte(`
...
 
   # Allow apps from the same package to signal each other via signals
-  signal peer=@{APP_PKGNAME}_*,
+  signal peer=@{SNAP_NAME}_*,
```
Comment (jdstrand) [inline interfaces/apparmor/template.go]: This needs to be:
`+  signal peer=snap.@{SNAP_NAME}.*,`

**Comment by jdstrand** [comment]
I'll let someone else comment on the code changes. The apparmor variable changes look fine assuming you make the suggested changes.

File: interfaces/apparmor/template.go (hunk 1 above)
Comment (zyga) [inline interfaces/apparmor/template.go]: Ah, I remember your comment the other day now. Thanks fixing

Comment (zyga) [inline interfaces/apparmor/template.go]: @jdstrand fixed in a481e20598d31351cbe0c7a654d360a81ff36e71

File: interfaces/apparmor/template.go (hunk 2 above)
Comment (zyga) [inline interfaces/apparmor/template.go]: @jdstrand fixed in a481e20598d31351cbe0c7a654d360a81ff36e71 (same as above)

**Comment by niemeyer** [comment]
Why do we not have integration test results anymore? :(

**Pull request merged**